web/source/resources/osk/kmwosk.css

```diff
@@ -16,18 +16,24 @@
 
 /* kmw-key-square applies only to OSK key elements, kmw-key-square-ex applies only to popup key elements */
 
+/* Common key-layout properties (all form factor + OS combinations) */
+
+.kmw-key-row {position: relative; width: 100%; height: 20%; margin: 0px; padding: 0px; border: 0px; overflow: hidden}
+.kmw-key-square {display: inline-block; position: relative; z-index: 10000; margin: 0px; border: 0px;
+  padding: 0px; box-sizing: border-box; -moz-box-sizing: border-box;}
+.kmw-key {display: block; position: relative; margin: 0px; text-align: center; box-sizing: border-box;
+  -moz-box-sizing: border-box;}
```

Comment:
There are notable internal changes for certain CSS class combinations here... but those changes make everything _way_ more consistent, which I believe to be a major plus.

Note that only properties related to the layout of keyboard elements were adjusted.  OSK _styling_ rules should be unaffected.

Reply:
* As we are making changes to the CSS, can we write out all rules in long form, e.g.

    ```css
    .kmw-key-row {
      position: relative;
      width: 100%;
      height: 20%;
      margin: 0px;
      padding: 0px;
      border: 0px;
      overflow: hidden
    }
    ```

* I don't think we need vendor-prefix on `box-sizing` (Firefox 29 or later supports it): ~~`-moz-box-sizing: border-box`~~

* It's good that you are splitting layout vs formatting here. Is `text-align` formatting rather than layout given it's about the content inside the key? I'd like to try and make those stylistic differences more tweakable in future theming.